core/app/models/spree/unit_cancel.rb

```diff
@@ -6,18 +6,18 @@ class Spree::UnitCancel < Spree::Base
   DEFAULT_REASON = 'Cancel'
 
   belongs_to :inventory_unit
-  has_one :adjustment, as: :source, dependent: :destroy
+  has_many :adjustments, as: :source, dependent: :destroy
```

Comment:
@bbuchalter good point.  I updated to @jhawthorn's solution though, so I've reverted back to `has_one`.